bolt/lib/Passes/ProfileQualityStats.cpp

```diff
@@ -365,6 +396,74 @@ void printCFGFlowConservationStats(raw_ostream &OS,
   }
 }
 
+void printExceptionHandlingStats(const BinaryContext &BC, raw_ostream &OS,
```

Comment:
I'm trying to understand what's collected by this function: is it the part of EC that's attributed to LP blocks and invokes?